ui/round/src/view/replay.ts

```diff
@@ -23,7 +23,6 @@ const scrollMax = 99999,
 
 const autoScroll = throttle(100, (movesEl: HTMLElement, ctrl: RoundController) =>
   window.requestAnimationFrame(() => {
-    if (ctrl.data.steps.length < 7) return;
```

Comment:
this is here to speed up the first moves of the games, when the JS JIT is cold and we don't need to autoscroll.

Instead of removing it you could check if the game is over.